wa/commands/create.py

```diff
@@ -143,19 +282,23 @@ def initialize(self, context):
     def execute(self, state, args):  # pylint: disable=R0201
         package_dir = args.path or os.path.abspath('.')
         template_path = os.path.join(TEMPLATES_DIR, 'setup.template')
-        self.create_extensions_package(package_dir, args.name, template_path, args.force)
+        self.create_extensions_package(
+            package_dir, args.name, template_path, args.force)
 
     def create_extensions_package(self, location, name, setup_template_path, overwrite=False):
         package_path = os.path.join(location, name)
         if os.path.exists(package_path):
             if overwrite:
-                self.logger.info('overwriting existing "{}"'.format(package_path))
+                self.logger.info(
+                    'overwriting existing "{}"'.format(package_path))
                 shutil.rmtree(package_path)
             else:
-                raise CommandError('Location "{}" already exists.'.format(package_path))
+                raise CommandError(
+                    'Location "{}" already exists.'.format(package_path))
         actual_package_path = os.path.join(package_path, name)
         os.makedirs(actual_package_path)
-        setup_text = render_template(setup_template_path, {'package_name': name, 'user': getpass.getuser()})
+        setup_text = render_template(
+            setup_template_path, {'package_name': name, 'user': getpass.getuser()})
```

Comment:
These sorts of changes that are nothing to do with the database should be in a separate commit.